collector/collector.go

```diff
@@ -295,35 +348,39 @@ func requiredEnv(envName string) string {
 	return ret
 }
 
-// handleMessage takes a message read from the kubeshark websocket and processes it for insertion
+// handleTrafficItem takes a message read from the kubeshark websocket and processes it for insertion
 // into the postgres database. It returns the processed message and a CASTMetadata struct for the message.
-func handleMessage(message []byte, msgStruct *Message) ([]byte, []byte, error) {
+func handleTrafficItem(trafficItem []byte, itemStruct *TrafficItem) ([]byte, []byte, error) {
```

Comment:
This naming is a little hard to follow, I would suggest calling the struct the trafficItem
```suggestion
func handleTrafficItem(messageJSON []byte, trafficItem *TrafficItem) ([]byte, []byte, error) {
```

Reply:
done